src/App.tsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
 
+import posts from './api/posts';
+import users from './api/users';
+import comments from './api/comments';
+import { PostList } from './components/PostList';
+import { Post } from './types/Post';
+
 import './App.scss';
 
-// import posts from './api/posts';
-// import comments from './api/comments';
-// import users from './api/users';
+const preparedPosts: Post[] = posts.map(post => {
+  const user = users.find(person => person.id === post.userId) || null;
+  const reviews = comments.filter(comment => comment.postId === post.userId);
```

Comment:
reviews and comments не всегда одно и тоже
ревью это оценка чего-то, комментарии часто не несут смысловую нагрузку)

я бы назвал все-таки comments
```suggestion
  const comments = comments.filter(comment => comment.postId === post.userId);
```